### DISCUSSION TITLE
Search in another column is search results are empty

### DISCUSSION THREAD

**Discussion by harshuchilzy** [rappasoft/laravel-livewire-tables#1261] [Feature Requests]
**Use case**: 
My Product table has few columns [id, name, price, sku, supplier]. Table search is enabled for name and sku. 
If somebody search by product name and results are empty, I want to lookup for the supplier column and show products likely to have suppliers for the search term.
Any help for this ?

**Comment by lrljoe** [comment]
For that you're going to have multiple queries firing which isn't ideal or something that is supported out of the box I'm afraid.

**Comment by lrljoe** [comment]
Please do raise an issue for a feature, and if it has much demand then it's likely to get prioritised!

**Reply by harshuchilzy** [reply]
I will, Thank you

**Reply by lrljoe** [reply]
There should be a "Feature Request" template in the Issues now, if you raise it in there then it can be prioritised etc!
